demo-terraform/aws/wordpress_ec2_alb_rds/alb.tf

```diff
@@ -1,53 +1,58 @@
 locals {
-    lb_name = "alb${random_string.lab_id.result}"
-    tg_name = "tg${random_string.lab_id.result}"
+  lb_name = "alb${random_string.lab_id.result}"
+  tg_name = "tg${random_string.lab_id.result}"
 }
 
 resource "aws_lb" "web" {
 
-    name = local.lb_name
-    internal = false ## External LB
+  name     = local.lb_name
+  internal = false ## External LB
 
-    load_balancer_type = "application"
-    ip_address_type = "ipv4"
-   
-    security_groups = [ aws_security_group.lb.id ]  ## Front end security group
-    subnets = module.vpc.public_subnets
+  load_balancer_type = "application"
+  ip_address_type    = "ipv4"
 
-    tags = {
-        Name = local.lb_name
-        "Lab-ID" = random_string.lab_id.result
-    }
+  security_groups = [aws_security_group.lb.id] ## Front end security group
+  subnets         = module.vpc.public_subnets
+
+  tags = {
+    Name     = local.lb_name
+    "Lab-ID" = random_string.lab_id.result
+  }
+
+  access_logs {
+    bucket  = "<s3_bucket_name>"
+    enabled = true
+  }
 }
 
 # For each target group, creates a target group and listener in the same port and protocol
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure one target group is configured to listen on HTTPS for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one target group listening on HTTPS ensures end-to-end encryption is not enabled. This could impact the confidentiality of data in transit.
 
**Remediation**
Elastic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: accurics-